_docs-2/apidocs/org/apache/accumulo/core/data/Condition.html

```diff
@@ -611,7 +611,7 @@ <h4>setIterators</h4>
 <div class="block">Set iterators to use when reading the columns value. These iterators will be applied in
  addition to the iterators configured for the table. Using iterators its possible to test other
  conditions, besides equality and absence, like less than. On the server side the iterators will
- be seeked using a range that covers only the family, qualifier, and visibility (if the
+ be sought using a range that covers only the family, qualifier, and visibility (if the
```

Comment:
This corrects the english, but I feel its not as descriptive of what it going on in the tserver.  What do you think about the following

```
On the server side `seek()` is called on the iterators using a range ...
```

I think this may be generated javadoc.  Could update it in the source code also.

Reply:
Ugh, just realized thats javadoc so can't use markdown, so maybe :

```
On the server side {@code seek()} is called on the iterators using a range ...
```

Reply:
Actually there may be an advantage to it being javadoc, could actually link to the seek function.

Reply:
Ohhh that's a good point I see it's generated by the javadoc. Thanks!

Reply:
Since this is generated, it's probably best to just fix the wording in the javadoc. I'm going to close this PR, with @milleruntime 's suggestion for `table_design.md` so the branch that GitHub automatically created when the file was edited from the web UI can be removed.